index.d.ts

```diff
@@ -0,0 +1,28 @@
+declare module 'objection-hashid' {
+  import { Model } from 'objection-2';
+  import { QueryBuilder } from 'knex';
+
+  class HashIdExtensionClass extends Model {
+    static get hashIdSalt(): string;
+
+    static get hashIdMinLength(): number | void;
+
+    static get hashIdAlphabet(): string | void;
+
+    static get hashIdSeps(): string | void;
+
+    static get _hashIdInstance(): string;
+
+    get hashId(): string;
+
+    get hashid(): string;
+
+    static get hashIdField(): string;
+
+    static get hashedFields(): Array<any>;
+
+    $formatJson(obj: object): object;
```

Comment:
We shouldn't have to override this as it's already a method of objection's built-in Model. We're just extending it.

Reply:
I remove line 24, assuming that is what you were referring to. As I said just let me know if I have made an error :sweat_smile: